src/components/Form.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import { useState } from "react";
```

Comment:
In my form component:

- `handleSubmit()` is quite different; we don't have the `trim()` check anymore.
- The events use `event` rather than `e`